index.ts

```
@@ -0,0 +1 @@
+function RRR() {}
```

Biome

[error] Unexpected empty block.

Empty blocks are usually the result of an incomplete refactoring. Remove the empty block or add a comment inside it if it is intentional.

(lint/suspicious/noEmptyBlockStatements)